src/applications/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -43,24 +41,18 @@ export function getEnrollmentData() {
             // EVSS partner service has no record of this user
             return dispatch({ type: NO_CHAPTER33_RECORD_AVAILABLE });
           }
-          return Promise.reject(
-            new Error(
-              `post-911-gib-status getEnrollmentData() received unexpected error: ${
-                error.status
-              }: ${error.title}: ${error.detail}`,
-            ),
+          Sentry.captureException(
+            `post-911-gib-status getEnrollmentData() received unexpected error: ${
+              error.status
+            }: ${error.title}: ${error.detail}`,
```

Comment:
Instead of rejecting with an `Error`, I've removed the outer `catch` and placed the `Sentry` call directly in here.